packages/sidecar/src/mcp/mcp.ts

```diff
@@ -12,7 +12,7 @@ import {
 import { getBuffer } from "~/utils/index.js";
 import { NO_ERRORS_CONTENT, NO_LOGS_CONTENT } from "./constants.js";
 
-export function createMcpInstance() {
+export function createMCPInstance() {
   const mcp = new McpServer({
```

Comment:
shouldn't we track this with our instrumentation?
`WrapMcpServerWithSentry?`

Reply:
Yes @MathurAditya724 shouldn't we track our stuff? 😀